lib/src/database/matrix_sdk_database.dart

```diff
@@ -153,10 +148,14 @@ class MatrixSdkDatabase extends DatabaseApi {
     this.database,
     this.idbFactory,
     this.sqfliteFactory,
-    this.maxFileSize = 0,
-    this.fileStoragePath,
-    this.deleteFilesAfterDuration,
-  });
+    int maxFileSize = 0,
+    Uri? fileStoragePath,
```

Comment:
I'd propose to include the migration I added in my initial PR - this keeps the change non-breaking.

Reply:
Sounds good to me. One moment...